queries.sql

```diff
@@ -116,4 +116,71 @@ SELECT owners.full_name, COUNT(animals.id) AS num_animals
 FROM owners
 LEFT JOIN animals ON owners.id = animals.owner_id
 GROUP BY owners.id
-ORDER BY num_animals DESC;
\ No newline at end of file
+ORDER BY num_animals DESC;
+
+-- Who was the last animal seen by William Tatcher?
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230588586-db6f7716-d6a7-4be2-b47c-14ceffce7d8b.png)

- [ ] It would be great if you add the highlighted part to your Getting Started section which you already have below in your readme file 🤔 After adding this information into Getting Started section please remove all the things in the image shown 👍 to make your readme file descriptive and professional 😊